WooCommerce/WooCommerceTests/ViewRelated/Survey/SurveyCoordinatorControllerTests.swift

```diff
@@ -43,7 +43,7 @@ final class SurveyCoordinatingControllerTests: XCTestCase {
         factory.surveyViewController.onCompletion()
 
         // Then
-        waitUntil {
+        waitUntil(timeout: 3.0) {
```

Comment:
just wondering, is this overriding the default timeout 5s because we know this call finishes sooner?

Reply:
Thanks for noticing that. I forgot to remove this when I changed the default timeout to 5.0 in 0f8cf76b6eb4e464b1f26811838ddcd34fb1a347. Fixed in 37129f819e8094f964965bf28be158af4d759b51.